### DISCUSSION TITLE
Adjusting beatmap's parameters if star rating>10 makes `BeatmapAttributesDisplay` "wobble"

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#25764] [Q&A] [answered]
https://github.com/ppy/osu/assets/86934170/49fd5af1-8a23-4648-bf9a-22284a2994f3

Colored box under the star rating should be fixed in size like if it was under 10.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/25772
